ui/app/src/components/ToolsPanelEmbeddedAppViewButton/ToolsPanelEmbeddedAppViewButton.tsx

```diff
@@ -56,14 +60,48 @@ const useStyles = makeStyles((theme) =>
 export default function ToolsPanelEmbeddedAppViewButton(props) {
   const [open, setOpen] = useState(false);
   const openEmbeddedApp = () => setOpen(true);
```

Comment:
Please check what happens when you...
1. open dialog
2. minimize it
3. click again on the tool panel item to open